plugins/cache-ehcache/src/main/java/org/opensearch/cache/store/disk/EhcacheDiskCacheManager.java

```diff
@@ -68,16 +76,21 @@ private EhcacheDiskCacheManager() {}
      * @param threadPoolAlias alias for disk thread pool
      * @return persistent cache manager
      */
-    public synchronized static PersistentCacheManager getCacheManager(
+    public static PersistentCacheManager getCacheManager(
         CacheType cacheType,
         String storagePath,
         Settings settings,
         String threadPoolAlias
     ) {
-        return cacheManagerMap.computeIfAbsent(
-            cacheType,
-            type -> new Tuple<>(createCacheManager(cacheType, storagePath, settings, threadPoolAlias), new AtomicInteger(0))
-        ).v1();
+        try {
```

Comment:
Is `getCacheManager` invoked at the start of every search request execution?

Reply:
No. Just once during node bootup.